main.py

```diff
@@ -1,7 +1,39 @@
+from django.contrib.contenttypes.fields import GenericRel  # noqa: F401
+
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+
+    for name in genres:
+        Genre.objects.create(
+            name=name
+        )
+    actors = [
+        ("George Klooney"),
+        ("Kianu Reaves"),
+        ("Scarlett Keegan"),
+        ("Will Smith"),
+        ("Jaden Smith"),
+        ("Scarlett Johansson"),
+    ]
+
+    for actor in actors:
+        first_name, last_name = actor.split(" ")
```

Comment:
Splitting the actor name assumes that each name consists of exactly two parts (first and last name). Ensure that all names in the `actors` list follow this format, or handle exceptions where names might not split into exactly two parts.